conformance/tests/conformance-rbac.yaml

```diff
@@ -27,6 +27,7 @@ rules:
   - deployments
   verbs:
   - create
+  - update
```

Comment:
is it really needed? does it mean it didn't work before?

Reply:
Yeah this was weird. All of a sudden I got errors that the tests couldn't update a deployment due to rbac permissions. I'll try running again to see what happens.

Reply:
Seems to just be an issue locally for me. Passing in the pipeline without the change.

Reply:
The first time I run the tests, they pass. Every time after that, I get the RBAC error.

Reply:
Maybe the test suite fails to clean up the deployments causing a create to become an update?